src/index.tsx

```diff
@@ -3,13 +3,24 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { BrowserRouter } from "react-router-dom";
+import { Routes } from "react-router-dom";
+import { Route } from "react-router-dom";
```

Comment:
通常我們會習慣把相同來源的寫在一起

```suggestion
import { BrowserRouter, Routes, Route } from "react-router-dom";
```

另外因為前端會用 [eslint](https://eslint.org/) 整理和檢查 code，他有一個很神的 plugin 叫做 [simple-import-sort](https://www.npmjs.com/package/eslint-plugin-simple-import-sort)，可以幫你做 import 的整理和排序，因此設定好 eslint＆VSCode 的 format on save 之後就可以再也不管它了（超香）

Reply:
我設定結束之後 eslint 好像不會自動幫我把他們合併～？

Reply:
等等直接用講的好了ＸＤ